test/common.jl

```diff
@@ -0,0 +1,21 @@
+using ODE, DiffEqBase, DiffEqProblemLibrary
```

Comment:
Guess I never did name it WIP, my bad. Should be complete now though, sans the ode23s issue which I think is beyond this PR.